project/ProjectDependencies.scala

```diff
@@ -22,6 +22,8 @@ object ProjectDependencyVersions {
   val Log4jVersion = "2.5"
   val ProtobufVersion = "2.5.0"
   val SparkVersion = "1.6.1"
+  val VertxVersion = "3.0.0"
+  val ExampleVertxVersion = "3.3.2"
```

Comment:
It is backwards compatible, but using 3.3.2 in the adapter also has the risk of using APIs not available in Vert.x 3.0.0, which would break the compatibility of the adapter, wouldn't it?